backend/deepchecks_monitoring/resources.py

```diff
@@ -393,6 +396,13 @@ def shutdown_parallel_check_executors_pool(self):
         import ray  # noqa
         ray.shutdown()
 
+    @tenacity.retry(
```

Comment:
why do we need the retry here?

Reply:
If it failes due to connection error or such